builtin/commit-graph.c

```diff
@@ -49,6 +49,8 @@ static int graph_verify(int argc, const char **argv)
 	struct stat st;
 	int flags = 0;
 
+	trace2_cmd_mode("verify");
```

Comment:
@jeffhostetler, @derrickstolee  Do we need this to be more granular? Shall I move it past the options parse and then set the mode as "shallow verify" and "full verify" based on if --shallow is passed in?